client/lib/invites/accept/index.jsx

```diff
@@ -8,11 +8,11 @@ import classNames from 'classnames';
 /**
  * Internal Dependencies
  */
-import InviteHeader from './invite-header';
-import LoggedInAccept from './logged-in-accept';
-import LoggedOutInvite from './logged-out-invite';
+import InviteHeader from 'components/invites/invite-header';
```

Comment:
We should avoid app-components in client/components (purer UI components). I'd place invite react components in "my-sites".